.github/workflows/benchmark.yml

```diff
@@ -119,7 +130,7 @@ jobs:
           ref: ${{github.event.inputs.repo_ref || 'main'}}
       - name: Copy repository - offline
         if: ${{inputs.offline_mode}}
-        run: cp -r /nvme/qa_test_models/offline_pkg/lmdeploy/* .
+        run: cp -r /nvme/qa_test_models/offline_pkg/lmdeploy/. .
```

Comment:
what's the motivation of changing `*` to `.`?

Reply:
cp -r /*  does not copy .github folder.  cp -r /. will copy .X file and folders too. It will fix this error
![image](https://github.com/InternLM/lmdeploy/assets/145004780/7451ac0f-c60d-48bd-ba24-624c66db0e6d)